src/libsync/account.cpp

```diff
@@ -1132,6 +1142,16 @@ void Account::setEncryptionCertificateFingerprint(const QByteArray &fingerprint)
     Q_EMIT wantsAccountSaved(sharedFromThis());
 }
 
+const QString &Account::synchronizationType() const
+{
+    return static_cast<AccessManager*>(_networkAccessManager.get())->synchronizationType();
+}
+
+void Account::setSynchronizationType(const QString &type)
+{
+    static_cast<AccessManager*>(_networkAccessManager.get())->setSynchronizationType(type);
```

Comment:
once this is set, each request will contain `SyncType: classical` or `SyncType: virtual`

I only see `SyncType: account` (from above) at app startup, after a proper sync that value will change through this method